pcsx2/CDVD/CdRom.cpp

```diff
@@ -552,7 +561,9 @@ void cdrReadInterrupt()
 
 	cdr.OCUP = 1;
 	SetResultSize(1);
-	cdr.StatP |= STATUS_READ | STATUS_ROTATING;
+	if (cdr.StatP & STATUS_SEEK)
+		cdr.StatP &= ~STATUS_SEEK;
```

Comment:
You don't really need to check if it's set to unset it, just unset it, it's probably more expensive to check the condition then unset it than just unset it every time.